language/move-prover/src/spec_translator.rs

```diff
@@ -46,6 +51,24 @@ pub struct SpecTranslator<'env> {
     name_to_idx_map: BTreeMap<Symbol, usize>,
 }
 
+impl<'env> SpecTranslator<'env> {
+    fn module_env(&self) -> &'env ModuleEnv<'env> {
+        use SpecEnv::*;
+        match self.spec_env {
+            Module(module_env) => module_env,
+            Function(func_target) => &func_target.func_env.module_env,
+        }
+    }
+
+    fn function_target(&self) -> &'env FunctionTarget<'env> {
+        use SpecEnv::*;
+        match self.spec_env {
+            Module(_) => unreachable!(),
```

Comment:
I would use panic! here as the code is not unreachable.

Reply:
Done.